--- origsrc/camljava-0.3/Makefile.config	2004-07-08 09:02:53.000000000 -0500
+++ src/camljava-0.3/Makefile.config	2008-01-08 21:01:40.968750000 -0600
@@ -11,6 +11,9 @@
 # Java compiler to use
 JAVAC=javac -g
 
+# Jar
+JAR=jar
+
 ################### JNI interface #####################
 
 ## Defaults are set for Sun's JDK 1.4.2 under Linux.  
--- origsrc/camljava-0.3/lib/Makefile	2001-06-21 06:46:05.000000000 -0500
+++ src/camljava-0.3/lib/Makefile	2008-01-08 21:01:49.453125000 -0600
@@ -11,7 +11,7 @@
 install:
 	mkdir -p $(CAMLJAVALIB)
 	cp jni.cma jni.cmi jni.cmxa jni.a libcamljni.a jni.mli $(CAMLJAVALIB)
-	jar cf $(CAMLJAVALIB)/camljava.jar fr/inria/caml/camljava/*.class
+	$(JAR) cf $(CAMLJAVALIB)/camljava.jar fr/inria/caml/camljava/*.class
 
 jni.cma: jni.cmo libcamljni.a
 	$(OCAMLC) -linkall -a -o jni.cma -custom jni.cmo \
--- origsrc/camljava-0.3/lib/jni.mlp	2003-02-20 07:42:40.000000000 -0600
+++ src/camljava-0.3/lib/jni.mlp	2008-01-08 20:58:21.203125000 -0600
@@ -22,6 +22,7 @@
   let sep =
     match Sys.os_type with
       "Unix" -> ":"
+    | "Cygwin" -> ":"
     | "Win32" -> ";"
     | _ -> assert false in
   let path =
